packages/sdk/src/lib/DeployUtil.ts

```diff
@@ -187,15 +344,43 @@ export const setSignature = (
   sig: ByteArray,
   publicKey: ByteArray
 ): Deploy => {
-  const signature = new Signature();
-  signature.setSigAlgorithm('ed25519');
-  signature.setSig(sig);
-
   const approval = new Approval();
-  approval.setApproverPublicKey(publicKey);
-  approval.setSignature(signature);
+  approval.signature = '01' + encodeBase16(sig);
```

Comment:
Please, create a proper function in `Keys.ts` and have the logic of `'01' + something`

Reply:
Done.